sddi/Dockerfile

```diff
@@ -1,107 +1,361 @@
-# #############################################################################
-# # Build stage
-# #############################################################################
-ARG BASEIMAGE_REPOSITORY=ghcr.io/tum-gis/ckan-sddi-base
-ARG BASEIMAGE_VERSION=latest
-ARG CKAN_VERSION_BUILD_STAGE=2.9.9-dev
-
-FROM ckan/ckan-base:${CKAN_VERSION_BUILD_STAGE} as extbuild
+###############################################################################
+# Extbuild stage
+###############################################################################
+FROM ckan/ckan-base:2.11.1 as extbuild
 
 USER root
 
-# ckanext-geoview #############################################################
-ARG CKANEXT_GEOVIEW_VERSION="v0.0.20"
-ENV CKANEXT_GEOVIEW_VERSION=${CKANEXT_GEOVIEW_VERSION}
-ENV CKANEXT_GEOVIEW_GITHUB_URL="https://github.com/ckan/ckanext-geoview"
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpq-dev \
+    gcc \
+    make \
+    g++ \
+    autoconf \
+    automake \
+    libtool \
+    patch \
+    musl-dev \
+    libpcre3-dev \
+    libpcre3 \
+    libffi-dev \
+    libxml2-dev \
+    libxslt-dev
+
+RUN pip install -U markupsafe==2.0.1 sqlalchemy==1.4.41
+
+# ckanext-hierarchy ###########################################################
+ARG CKANEXT_HIERARCHY_VERSION="abb4e2d"
+ENV CKANEXT_HIERARCHY_VERSION=${CKANEXT_HIERARCHY_VERSION}
+
+RUN set -ex && \
+    mkdir -p /wheels && \
+    pip install -r https://raw.githubusercontent.com/ckan/ckanext-hierarchy/${CKANEXT_HIERARCHY_VERSION}/dev-requirements.txt
+
+RUN set -ex && \
+  pip wheel --wheel-dir=/wheels -r https://raw.githubusercontent.com/ckan/ckanext-hierarchy/${CKANEXT_HIERARCHY_VERSION}/requirements.txt && \
+  pip wheel --wheel-dir=/wheels git+https://github.com/ckan/ckanext-hierarchy.git@${CKANEXT_HIERARCHY_VERSION}#egg=ckanext-hierarchy && \
+  curl -o /wheels/ckanext-hierarchy.txt https://raw.githubusercontent.com/ckan/ckanext-hierarchy/${CKANEXT_HIERARCHY_VERSION}/requirements.txt && \
+  ls -lah /wheels
+
+# ckanext-envvars
+ENV ENVVARS_GIT_URL=https://github.com/ckan/ckanext-envvars/
+ENV ENVVARS_GIT_BRANCH=v0.0.6
+
+RUN set -ex && \
+  pip3 wheel --wheel-dir=/wheels git+${ENVVARS_GIT_URL}@${ENVVARS_GIT_BRANCH}#egg=ckanext-envvars
+
+# ckanext-relation ############################################################
+ARG CKANEXT_RELATION_VERSION="1.1.0"
+ENV CKANEXT_RELATION_VERSION=${CKANEXT_RELATION_VERSION}
```

Comment:
Those extensions are already build and added in the base image. Check for all extensions, that they are only build and installed in either `base` or `sddi` variant.